forklift/values.yaml

```diff
@@ -0,0 +1,27 @@
+replicaCount: 1
+
+image:
+  repository: smartcitiesdata/forklift
+  tag: development
+  pullPolicy: Always
+
+kafka:
+  #comma delimited list of kafka seed brokers
+  brokers: streaming-service-kafka-bootstrap.streaming-prime:9092
+  strimziCluster: streaming-service
+  dataTopicPrefix: "transformed"
+  outputTopic: "streaming-persisted"
+
+nameOverride: ""
+fullnameOverride: ""
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+kdp:
```

Comment:
What did we land on with generic v specific overrides for these things? I thought that Jake, et. al. came up with a good standard in `streisand-deploy` where these have good documentation, and then in the deploy repo you'd make sure to set all of those to the scos specific values.

Reply:
Good thought. We've extracted the smart os deploy specific values to the overrides